### ISSUE TITLE
Inconsistent subfolder name for azure-eventhub module

### ISSUE THREAD

**Issue by glaubitz** [Azure/azure-sdk-for-python#7496]
The module ```azure-eventhub``` is located in the subfolder ```eventhub/azure-eventhubs```, so the module subdirectory name is not consistent with the module name:

```
glaubitz@suse-laptop:~/upstream/azure-sdk-for-python/sdk/eventhub> grep PACKAGE_NAME azure-eventhubs/setup.py 
# Change the PACKAGE_NAME only to change folder and different name
PACKAGE_NAME = "azure-eventhub"
package_folder_path = PACKAGE_NAME.replace('-', '/')
namespace_name = PACKAGE_NAME.replace('-', '.')
    name=PACKAGE_NAME,
glaubitz@suse-laptop:~/upstream/azure-sdk-for-python/sdk/eventhub> ls
azure-eventhubs  azure-mgmt-eventhub  ci.yml  tests.yml
glaubitz@suse-laptop:~/upstream/azure-sdk-for-python/sdk/eventhub>
```

**Comment by kaerm** [comment]
Hi @glaubitz thanks for letting us know, we will look into that cc: @yunhaoling

**Comment by annatisch** [comment]
Will be resolved by PR #8813

**Comment by annatisch** [comment]
Resolved
